core-bundle/src/Resources/contao/models/ContentModel.php

```diff
@@ -432,6 +433,42 @@ public static function countPublishedByPidAndTable($intPid, $strParentTable, arr
 
 		return static::countBy($arrColumns, array($intPid, $strParentTable), $arrOptions);
 	}
+
+	/**
+	 * Get the default meta data or null if not applicable.
+	 */
+	public function getOverwriteMetaData(): ?MetaData
+	{
+		// Ignore if `overwriteMeta` isn't set
+		if ('' === $this->overwriteMeta)
```

Comment:
```suggestion
		if (!$this->overwriteMeta)
```
Because `$this->overwriteMeta` can be null too (e.g. when used in the gallery module).